app/src/main/java/io/appium/uiautomator2/model/api/gestures/PinchModel.java

```diff
@@ -0,0 +1,15 @@
+package io.appium.uiautomator2.model.api.gestures;
+
+import io.appium.uiautomator2.model.RequiredField;
+import io.appium.uiautomator2.model.api.BaseModel;
+import io.appium.uiautomator2.model.api.ElementModel;
+
+public class PinchModel extends BaseModel {
+    public ElementModel origin;
+    public RectModel area;
+    @RequiredField
+    public Float percent;
+    public Integer speed;
+
+    public PinchModel() {}
```

Comment:
Codacy found an issue: [Avoid unnecessary constructors - the compiler will generate these for you](https://app.codacy.com/manual/dpgraham/appium-uiautomator2-server/pullRequest?prid=6222243)